### PR TITLE
feat: remove run function start container

### PR DESCRIPTION
**Pull request by jabrown85**
Product has requested the removal of this docker-based command. Users are redirected to use the run function start command instead.

### What does this PR do?

Removes the `sf run function start container` command.

### What issues does this PR fix or reference?

The command in question is a docker variation of the local run command and product has requested to remove the container version.

[W-12514513](https://gus.lightning.force.com/a07EE00001KunfCYAR)

A followup PR will be pushed to heroku/functions-core to remove the `benny` binary once usage is removed in this PR.

### CODE REVIEW

**Review by joeybrown-sf — APPROVED** [review]

**Review by edmorley — COMMENTED** [review]
There are a few container related leftovers on the top level `sf run function start` command too, here:
https://github.com/salesforcecli/plugin-functions/blob/b4194d6c1280c0cdd4c38e855ed435a182c12137/src/commands/run/function/start.ts#L18-L21

In fact, with those container-related args removed, the `Start` subclass used for `sf run functions start` could probably just use the base `Local` class's `flags` definition, and not need to duplicate `flags` itself?

Though up to you whether you'd prefer to leave that for future PRs :-)

**Review by edmorley — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -637,40 +638,9 @@ EXAMPLES
 
 ## `sf run function start container`
```
Comment (mimen) [inline README.md:639]: I notice the command still exists in order to inform the user that it has been deprecated. Maybe we should note that in the README? Or is this autogenerated?

**Review by mimen — APPROVED** [review]
Looks good!

File: README.md (hunk 1 above)
Comment (jabrown85) [reply README.md:639]: README is auto-generated - there is probably a way for me to add something here if you thought it was worth it. I'm fairly unfamiliar with all the CLI bits 😄

**Pull request merged**